### PR TITLE
Moving zeroes to end of array

### PR DESCRIPTION
**Pull request by HariniJeyaraman**
**Fixes issue:**

#4742

**Changes:**

 Moving zeroes to the end of array in C++

### CODE REVIEW

**Comment by HariniJeyaraman** [comment]
@arnavb Please review the code

File: move_zeroes_to_end.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <stdlib.h>
```
Comment (arnavb) [inline move_zeroes_to_end.cpp]: Move this code to the correct folder. Use `cstdlib`. In all for loops, use preincrement.

File: move_zeroes_to_end.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    for (int i = 0; i < n; i++)
+        std::cout << arr[i] << " ";
+}
+
+int main() {
+    int n;
+    std::cout << "Enter the size of array : \n";
+    std::cin >> n;
+    int a[n], arr[n];
```
Comment (arnavb) [inline move_zeroes_to_end.cpp]: This is non standards compliant C++. Use `std::vector` instead.

**Comment by arnavb** [comment]
I would put this code in the unclassified algorithms folder, since I can't think of a better place. Let me know if you have a better idea.

**Comment by arnavb** [comment]
Accidently closed.

**Comment by arnavb** [comment]
The filename should match the folder name exactly.

**Comment by HariniJeyaraman** [comment]
@arnavb Please review

**Pull request merged**